test/e2e/kubelet_perf.go

```diff
@@ -184,7 +184,7 @@ var _ = Describe("Kubelet [Serial] [Slow]", func() {
 			})
 		}
 	})
-	Describe("experimental resource usage tracking", func() {
+	Describe("experimental resource usage tracking [Feature:ExperimentalResourceUsageTracking]", func() {
```

Comment:
since this test is marked `[Slow]`, I think it's already skipped on gke-ci. Do we need to label it as a feature, too?